tests/schema.test.ts

```diff
@@ -207,3 +207,35 @@ test('it exposes findOne and findMany', async () => {
 
   expect(printSchema(schema)).toMatchSnapshot()
 })
+
+test('enum can be filtered on', async () => {
+  const datamodel = `
+    enum EA {
+      A
+      B
+      C
+    }
+    model MA {
+      fa Int @id
+      fb EA
+    }
+  `
+  const Query = queryType({
+    definition(t: any) {
+      t.crud.ma()
+      t.crud.mas()
+    },
+  })
+  const MA = objectType({
+    name: 'MA',
+    definition(t: any) {
+      t.model.fa()
+      t.model.fb()
+    },
+  })
+  const EA = enumType({ name: 'EA', members: ['A', 'B', 'C'] })
```

Comment:
@Weakky you mentioned this shouldn't be needed any more right? But if we remove this, the test will fail.